index.html

```diff
@@ -3,9 +3,58 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>myFirstProject2</title>
+    <title>jooble</title>
+    <link rel="stylesheet" href="./stylesheet.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
 </head>
 <body>
-    
+    <section class="top-row">
+        <header class="title">
+            <h1>Jooble</h1>
+        </header>
+        <div class="type-box">
+            <div class="keywords">Keywords</div>
+        </div>
+        <div class="type-box">
+            <div class="location">Location</div>
+        </div>
+        <div class="search-box">
+            <div class="search">Find Jobs</div>
+        </div>
+        <div class="login-box">
+            <div class="login">Log in</div>
+        </div>
+    </section>
+
+    <section class="left-panel">
+        <div class="filters-box-list">
+            <div class="filters-box">
+                <div class="filters">Filters</div>
+            </div>
+            <div class="dateOfPosting-box">
+                <div class="dateOfPosting">Date of posting</div>
```

Comment:
Tu tak samo - `class` to nie `id/nazwa` elementu. Możesz to grupować na elementy, które powtarzają się wyglądem.
Dotyczy to też elementów niżej.